GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -0,0 +1,156 @@
+
+    * Configure *Infrastructure Pipeline Variables*
+        * Select Pipelines tab from within side-navigation menu then select Library tab
+        * Click [+Variable group] and add the following variables:
+
+            | Name   | Value | Description
+            |-------------|-----------|-----------|
+            | `AGENT_POOL` | Hosted Ubuntu 1604 | The type of build agent used for your deployment. |
+            | `ARM_PROVIDER_STRICT` | false | Terraform ARM provider modification |
+            | `BUILD_ARTIFACT_NAME` | drop | Name to identity the folder containing artifacts output by a build. |
+            | `GO_VERSION`| 1.12.5 | The version of Go terraform deployments are bound to. |
+            | `PIPELINE_ROOT_DIR` | 'devops/providers/azure-devops/templates/' | A path for finding Cobalt templates. |
+            | `REMOTE_STATE_CONTAINER` | az-hw-remote-state-container | The remote storage container name for managing the state of a cobalt template's deployed infrastructure. Also is used as a naming convention for branching state into multiple workspaces. |
+            | `SCRIPTS_DIR` | infrastructure/scripts | Path to scripts used at runtime for composing build and release jobs at various pipeline stages. |
+            | `TEST_HARNESS_DIR` | test-harness/ | A path to the cobalt test harness for running integration and unit tests written in Docker and Golang. |
+            | `TF_ROOT_DIR`| infra | The primary path for all Cobalt templates and the module they are made of.  |
+            | `TF_VERSION`| 0.12.4 | The version of terraform deployments are bound to. |
+            | `TF_WARN_OUTPUT_ERRORS`| 1 | The severity level of errors to report |
+
+    > Important: Every targeted environment specified within the build pipeline expects a
+    > variable group specified with the naming convention `<ENVIRONMENT_NAME> Pipeline Variables`
+
+    * Configure *DevInt Pipeline Variables*
+        * Environment-specific variables have no default values and must be assigned
+        * Return to the Library tab
+        * Click [+Variable group] and add the following variables:
+
+            | Name  | Description
+            |-------------|-----------|
+            | `ARM_SUBSCRIPTION_ID` | The Azure subscription ID for the DevInt environment to which resources will be deployed |
+            | `REMOTE_STATE_ACCOUNT` | The remote state account name used to manage the state of this environment's deployed infrastructure |
+            | `SERVICE_CONNECTION_NAME` | The name of the service endpoint or connection created in a previous step with the Service Principal permissions to deploy resources to the specified Azure subscription |
+
+    * Configure *QA Pipeline Variables*
+        * Environment-specific variables have no default values and must be assigned
+        * Return to the Library tab
+        * Click [+Variable group] and add the same three variables with QA environment values
```

Comment:
This is confusing. Because we're deploying everything to the same environment, all values are the same. They should be different, but that would mean different service principal / connection, subscription ID, storage accounts, etc.